app/controllers/users_controller.rb

```diff
@@ -995,18 +995,21 @@ def destroy
     render json: success_json
   end
 
-  def ignore
+  def track
```

Comment:
I think this should be changed to `notification_level` to maintain consistency with "Topic Notification Levels".

Plus we're not really "tracking" any users here. Quite the opposite actually 😉

Reply:
Good suggestion, changed it 👍